### PR TITLE
feat: add resources for documentation driven development

### PR DESCRIPTION
**Pull request by shanejonas**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,13 +35,13 @@ For small documentation changes and fixes, these can be done quickly following t
 
 ### Cloning (or forking) the repository
 
-First off, you are going to need your own copy of the repository. You can find help on how to fork a repo [here](https://help.github.com/articles/fork-a-repo/).
+First off, find help on how to fork a repo [here](https://help.github.com/articles/fork-a-repo/).
```
Comment (stevanlohja) [inline CONTRIBUTING.md]: The heading here sounds like we are being inclusive of `cloning`, but I'd assume we want contributors to `fork` - almost always. Unless I'm mistaken, maybe we can remove the "Cloning" part of the heading to just leave "Forking the repository"

Comment (shanejonas) [reply CONTRIBUTING.md]: yea definitely want to encourage forking to then make PR's back. cloning doesn't really promote that

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,13 +35,13 @@ For small documentation changes and fixes, these can be done quickly following t
...
 
-### Submitting your changes
+### Submitting changes
 
-1. Review & Test your changes
+1. Review & Test changes
 
-If it's code: make sure you test it. If it's documentation: make sure you view it in a Markdown viewer or the like.
+If it's code: test it. If it's documentation: make sure to view it in a Markdown viewer or the like.
```
Comment (meowsbits) [inline CONTRIBUTING.md]:
```suggestion
If it's code: test it. If it's documentation: preview the rendered Markdown.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -51,7 +55,7 @@ Make sure you've signed the repository's Contributor License Agreement. We are n
  
 4. Submit a pull request
 
-Push your local changes to you forked repository and make a pull request. Follow the [Convention Commits](CONVENTIONAL_COMMITS.md) guidelines for naming Github pull requests and what to put in the body.
+Push local changes to you forked repository and make a pull request. Follow the [Convention Commits](CONVENTIONAL_COMMITS.md) guidelines for naming Github pull requests and what to put in the body.
```
Comment (meowsbits) [inline CONTRIBUTING.md]:
```suggestion
Push local changes to your forked repository and make a pull request. Follow the [Convention Commits](CONVENTIONAL_COMMITS.md) guidelines for naming Github pull requests and what to put in the body.
```

Comment (meowsbits) [reply CONTRIBUTING.md]: Is the idea here that a commit subject line should be reused as a PR title? (I don't see any mention of PR titling in that doc...)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,6 +2,10 @@
 
 > This document is inspired by [elasticsearch/CONTRIBUTING.md](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md)
 
+Adding a `CONTRIBUTING.md` to a github repository enables a link to that file in the pull request or create an issue page. This shows potential contributors how to make a meaningful impact on the project. Contribution guidelines save time and hassle caused by improper pull requests and issues. You can learn more about the features that are enabled by github when this file is present [here](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)
```
Comment (meowsbits) [inline CONTRIBUTING.md]:
```suggestion
Adding a `CONTRIBUTING.md` to a github repository enables a link to that file in the pull request or create an issue page. This document should guide potential contributors toward making a successful and meaningful impact on the project, and can save maintainers time and hassle caused by improper pull requests and issues. You can learn more about the features that are enabled by github when this file is present [here](https://help.github.com/articles/setting-guidelines-for-repository-contributors/).
```

Comment (shanejonas) [inline CONTRIBUTING.md]:
```suggestion
Adding a `CONTRIBUTING.md` to a github repository enables a link to that file in the pull request or create an issue page. This shows potential contributors how to make a meaningful impact on the project. Contribution guidelines save time and hassle caused by improper pull requests and issues. You can learn more about the features that are enabled by github when this file is [here](https://help.github.com/articles/setting-guidelines-for-repository-contributors/)
```

File: README.md [hunk 5]
```diff
@@ -1,11 +1,18 @@
...
+There are many ways to drive development but with open source specifically, starting at the README gives a middle ground between technical specifications and non-technical. This promotes open source contribution and engagement.
+
+> By the same principle a beautifully crafted library with no documentation is also damn near worthless. If your software solves the wrong problem or nobody can figure out how to use it, there’s something very bad going on.
+
+- [Readme Driven Development](http://tom.preston-werner.com/2010/08/23/readme-driven-development.html) by Tom Preson-Werner
 
 ## Getting Started
 
-To get started, you will need a [fork of the repository](https://help.github.com/articles/fork-a-repo/). Then you can edit this file and delete everything above this line.
+To get started, [fork](https://help.github.com/articles/fork-a-repo/) or [duplicate](https://help.github.com/articles/duplicating-a-repository/) the repository. Then edit this file and delete everything above this line.
```
Comment (meowsbits) [inline README.md:33]:
```
[...] Then edit this file and delete everything above this line.
```

Should the following paragraph `Conventions and Specifications` also be deleted?

File: CONTRIBUTING.md (hunk 4 above)
Comment (shanejonas) [reply CONTRIBUTING.md]: yea this sounds better

File: README.md (hunk 5 above)
Comment (shanejonas) [reply README.md:33]: Agreed

**Comment by meowsbits** [comment]
Nitpick: should `s/github/Github/g` since a proper noun?

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -31,13 +35,13 @@ For small documentation changes and fixes, these can be done quickly following t
 
 ### Cloning (or forking) the repository
```
Comment (shanejonas) [inline CONTRIBUTING.md]:
```suggestion
### Cloning the repository
```

**Comment by meowsbits** [comment]
Why do some files get `.md` extensions (eg. `CONTRIBUTING.md`) and others not get them (eg. `BUILDING`)?

File: CONTRIBUTING.md (hunk 6 above)
Comment (stevanlohja) [inline CONTRIBUTING.md]:
```suggestion
### Forking the repository
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -31,13 +35,13 @@ For small documentation changes and fixes, these can be done quickly following t
...
+First off, find help on how to fork a repo [here](https://help.github.com/articles/fork-a-repo/).
 
-### Submitting your changes
+### Submitting changes
 
-1. Review & Test your changes
+1. Review & Test changes
 
-If it's code: make sure you test it. If it's documentation: make sure you view it in a Markdown viewer or the like.
+If it's code: test it. If it's documentation: preview the rendered Markdown.
```
Comment (stevanlohja) [inline CONTRIBUTING.md]:
```suggestion
If changes in code, then test the code. If changes in documentation, then preview the rendered Markdown.
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (stevanlohja) [inline CONTRIBUTING.md]:
```suggestion
How to fork a repo [here](https://help.github.com/articles/fork-a-repo/).
```

**Comment by shanejonas** [comment]
@meowsbits agreed. fixed all

File: README.md [hunk 8]
```diff
@@ -1,11 +1,18 @@
 # Pristine
 
-Pristine is an open source repository in its original condition. It is meant to be a starting point and a place to find resources to augment your own open source repositories.
+Pristine is an open source repository in its original condition. It's goal is to be a starting point for open source repositories wanting to follow Documentation Driven Development. It is also a place for existing projects to find resources to augment current documentation.
```
Comment (stevanlohja) [inline README.md]:
```suggestion
Pristine is an open source repository in its original condition. Its goal is to be a starting point for open source repositories wanting to follow Documentation Driven Development. It is also a place for existing projects to find resources to augment current documentation.
```

File: README.md [hunk 9]
```diff
@@ -1,11 +1,18 @@
...
 
+## Documentation Driven Development
+
+There are many ways to drive development but with open source specifically, starting at the README gives a middle ground between technical specifications and non-technical. This promotes open source contribution and engagement.
```
Comment (stevanlohja) [inline README.md]:
```suggestion
There are many ways to drive open source development, starting at the README gives a middle ground between technical specifications and non-technical. This promotes open source contribution and engagement.
```

**Review by stevanlohja — CHANGES_REQUESTED** [review]
I've left some `suggested changes`

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -29,15 +33,15 @@ There are many ways to fix a problem and it is important to find the best approa
 
 For small documentation changes and fixes, these can be done quickly following this video guide on [how to contribute to Open Source in 1 minute on Github](https://www.youtube.com/watch?v=kRYk1-yKwWs).
 
-### Cloning (or forking) the repository
+### Forking the repository
 
-First off, you are going to need your own copy of the repository. You can find help on how to fork a repo [here](https://help.github.com/articles/fork-a-repo/).
+First off, find help on how to fork a repo [here](https://help.github.com/articles/fork-a-repo/).
```
Comment (shanejonas) [inline CONTRIBUTING.md]:
```suggestion
[How to fork a repository](https://help.github.com/articles/fork-a-repo/).
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -29,15 +33,15 @@ There are many ways to fix a problem and it is important to find the best approa
...
 
-### Submitting your changes
+### Submitting changes
 
-1. Review & Test your changes
+1. Review & Test changes
 
-If it's code: make sure you test it. If it's documentation: make sure you view it in a Markdown viewer or the like.
+If it's code: test it. If it's documentation: preview the rendered Markdown.
```
Comment (shanejonas) [inline CONTRIBUTING.md]:
```suggestion
If the code changed, then test it. If documentation changed, then preview the rendered Markdown.
```

**Review by stevanlohja — APPROVED** [review]

**Pull request merged**